recipe/meta.yaml

```diff
@@ -52,14 +53,20 @@ requirements:
 test:
   requires:
     - {{ compiler('c') }}
+  imports:
+    - vapoursynth
```

Comment:
Didn't conda automatically import this? If not, how did
```
"$PYTHON" -c "from vapoursynth import core; print(core.version())"
```
pass the test?

Reply:
Actually, the imports in `test` part in meta.yaml, the conda-build package will run the python -c "import vapoursynth" automatically.

Reply:
I see. I used `core.version()` to test the package since Vapoursynth automatically loads plugins under `core` and I was concerned whether the plugins could be correctly found. This is slightly different from simply `import vapoursynth`. What do you think?